src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -63,6 +63,38 @@ public string GetStringVariation(string key, ICurrentContext currentContext, str
         return _client.StringVariation(key, BuildContext(currentContext), defaultValue);
     }
 
+    public Dictionary<string, object> GetAll(ICurrentContext currentContext)
+    {
+        var results = new Dictionary<string, object>();
+
+        var keys = FeatureFlagKeys.GetAllKeys();
+
+        var values = _client.AllFlagsState(BuildContext(currentContext));
```

Comment:
Seems a little odd to me that none of these client requests are `async`. Other methods in this service, too.

Reply:
I agree -- not sure why their SDK has no `async` support since it's quite stable and has many major versions.